src/views/DatasetManager/components/CreateDatasetButton/CreateSubDatasetButton.js

```diff
@@ -43,5 +43,5 @@ export const CreateSubDatasetButton = ({ parentDatasetId }) => {
 };
 
 CreateSubDatasetButton.propTypes = {
-  parentDatasetId: PropTypes.string,
+  parentDataset: PropTypes.object,
```

Comment:
The component should handle the cases when this prop is undefined without crashing. Also this case should never happen (I think the button is only visible in `DatsetOverview`, when a dataset is selected)

We could probably disable the button when `parentDataset` is undefined to be consistent with several other buttons, but this change is low-priority (because the case will never happen :slightly_smiling_face:)